cypress/support/step-definitions/button-toggle-steps.js

```diff
@@ -68,3 +68,7 @@ Then('Button Toggle is not grouped', () => {
   buttonTogglePreview().eq(SECOND_ELEMENT).should('have.css', 'margin-left', '10px');
   buttonTogglePreview().eq(THIRD_ELEMENT).should('have.css', 'margin-left', '10px');
 });
+
+When('I click on Button Toggle {int}', (button) => {
```

Comment:
If you use `int` in step, so `index` in brackets will be more readable 
```suggestion
When('I click on Button Toggle {int}', (index) => {
```

Reply:
👍